<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-IG5V1RQ" failures="0" tests="5" name="databaseTesting.MyTestcases" time="0.633" errors="1" timestamp="2024-11-20T17:30:35 EET" skipped="4">
  <testcase classname="databaseTesting.MyTestcases" name="mySetup" time="0.633">
    <error message="Access denied for user &#039;root&#039;@&#039;localhost&#039; (using password: YES)" type="java.sql.SQLException">
      <![CDATA[java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:121)
at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:114)
at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:837)
at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:420)
at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:238)
at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:180)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:681)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
at databaseTesting.MyTestcases.mySetup(MyTestcases.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- mySetup -->
  <system-out/>
  <testcase classname="databaseTesting.MyTestcases" name="AddDataToMyDataBase" time="0.000">
    <skipped/>
  </testcase> <!-- AddDataToMyDataBase -->
  <system-out/>
  <testcase classname="databaseTesting.MyTestcases" name="Updatedata" time="0.000">
    <skipped/>
  </testcase> <!-- Updatedata -->
  <system-out/>
  <testcase classname="databaseTesting.MyTestcases" name="ReadData" time="0.000">
    <skipped/>
  </testcase> <!-- ReadData -->
  <system-out/>
  <testcase classname="databaseTesting.MyTestcases" name="DeleteData" time="0.000">
    <skipped/>
  </testcase> <!-- DeleteData -->
  <system-out/>
</testsuite> <!-- databaseTesting.MyTestcases -->
